scripts/config_fullnat_2arm_sample.sh

```diff
@@ -0,0 +1,146 @@
+#!/bin/sh
+#$Date: 2018-01-19
+#$ID: icymoon
+
+##### Basic configurations #####
+##### NOTE: modify these according to your environment #####
+WAN_IP_LIST=('192.168.10.2' '192.168.10.4' '192.168.10.5') 
+WAN_PORT=80
+WAN_GW='192.168.10.0/24'
```

Comment:
"Gateway" should not be `.0`, you can use `WAN_NET='192.168.10.0/24'`. And the same with `LAN_GW`.